data/json/items/melee/swords_and_blades.json

```
@@ -1554,6 +1569,72 @@
     "flags": [ "DURABLE_MELEE", "SHEATH_SWORD" ],
     "weapon_category": [ "MEDIEVAL_SWORDS", "SHORT_SWORDS" ],
     "category": "weapons"
+  },
+    {
+    "id": "shortsword",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": { "str": "shortsword" },
+    "description": "A sword, larger than a dagger, but shorter than an Arming Sword, and the perfect length for a sidearm.  Despite its diminuative size compared to other swords, it's still a deadly weapon.",
+    "price": 100000,
+    "price_postapoc": 4500,
+    "material": [ "steel" ],
+    "flags": [ "DURABLE_MELEE", "SHEATH_SWORD" ],
+    "techniques": [ "WBLOCK_2" , "RAPID" ],
+    "weight": "900 g",
+    "volume": "1500 ml",
+    "longest_side": "50 cm",
+    "bashing": 3,
+    "cutting": 28,
+    "to_hit": { "grip": "weapon", "length": "short", "surface": "line", "balance": "good" },
+    "category": "weapons",
+    "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 10 ] ],
+    "weapon_category": [ "MEDIEVAL_SWORDS", "MEDIUM_SWORDS" ]
```

```suggestion
    "weapon_category": [ "MEDIEVAL_SWORDS", "SHORT_SWORDS" ]
```
I'm having a big think 🤔